app/models/order.rb

```diff
@@ -128,6 +131,15 @@ def delete_orders_packages
     end
   end
 
+  def international_orders?
+    detail_type == "Shipment" || detail_type == "CarryOut"
```

Comment:
```suggestion
    [Type::SHIPMENT, Type::CARRY_OUT].includes?(detail_type)
```

and ORDER_TYPE can be a module in the order model something like
```
   module Type
      SHIPMENT = 'Shipment'
      CARRY_OUT = 'CarryOut'
   end
```

Reply:
So u can make use of those type anywhere in the application just by doing
`Order::Type::SHIPMENT` or `Order::Type::CARRY_OUT`